integration-tests/gatsby-image/package.json

```diff
@@ -11,7 +11,7 @@
     "gatsby-plugin-offline": "next",
     "gatsby-plugin-react-helmet": "next",
     "gatsby-plugin-sharp": "^2.0.0-rc.7",
-    "gatsby-source-filesystem": "^1.5.39",
+    "gatsby-source-filesystem": "next",
```

Comment:
Should we just use `next` for all of these deps? And then not commit lock files?

Reply:
we can revisit if we will use `next (soon to be `latest`) or caret version thingie for tests soon - this is just quick'n'dirty fix to get tests running and hopefully passing ;)